src/redis/index.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * External dependencies
+ */
+const IORedis = require( 'ioredis' );
+
+/**
+ * Internal dependencies
+ */
+let { logger } = require( '../logger/index' );
+
+// Create a class in case we need to scale the connections
+class Redis {
+	constructor() {
+		const [ host, port ] = ( process.env.REDIS_MASTER || '' ).split( ':' );
```

Comment:
I think this needs to move inside the `else`, i.e. after we check if `process.env.REDIS_MASTER` is already there.

Reply:
Ah right, moved 👍